environment.go

```diff
@@ -82,6 +82,27 @@ func (e *Environment) Delete(key string) {
 	delete(e.data, key)
 }
 
+// Clear will delete all key value pairs from the environment
+//
+// Example:
+//
+//	env := NewEnvironment(t)
+//	env.Put("key1", 123)
+//	env.Put("key2", 456)
+//	env.Clear()
+func (e *Environment) Clear() {
+	opChain := e.chain.enter("Clear Data")
+	defer opChain.leave()
+
+	e.mu.Lock()
+	defer e.mu.Unlock()
+
+	for key := range e.data {
+		delete(e.data, key)
+	}
```

Comment:
IIRC, this way of clearing map will keep allocated buckets inside the map.

Suppose the user have a few large groups of tests. Each group clears environment first and then tests in this group extensively use environment.

In this case, I think it's more convenient to allow GC to fully collect environment memory from previous tests, to avoid unnecessary heap growth.

Long story short, I suggest to replace this with:

```go
e.data = make(map[string]interface{})
```

:)

Reply:
This makes perfect sense. Thank you for clearing this up, and I will make sure to make the change.